extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -263,6 +288,10 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   onPlpCardClick_(shoppingData) {
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.STORY_SHOPPING_PRODUCT_ID,
+      shoppingData?.productId
+    );
```

Comment:
Do we need this here? Aren't we handling all of this logic in `updateTemplate_`?

Reply:
Yes, it's not needed, removed.